File: ProxyPattern/python/Client/Proxy.py
import abc

class Proxy(metaclass=abc.ABCMeta):
    @abc.abstractmethod
    def addBalanceAccount(self, balanceAccount:int):
        return NotImplemented
    
    @abc.abstractmethod
    def subtractBalanceAccount(self, balanceAccount:int):
        return NotImplemented
    
    @abc.abstractmethod
    @property
    def balanceAccount(self) -> int:
        return NotImplemented

File: ProxyPattern/java_/Proxy.java
package ProxyPattern.java_;

import java.rmi.Remote;
import java.rmi.RemoteException;

public interface Proxy extends Remote{
    void addBalanceAccount(int balanceAccount) throws RemoteException;
    void subtractBalanceAccount(int balanceAccount) throws RemoteException;
    int getBalanceAccount() throws RemoteException;
}
